lib/sync.js

```diff
@@ -36,7 +36,25 @@ var DEBUG = true;
 // beyond that and wedge forever, so we need to track how long we are willing
 // to keep open the connection. This constant is *ADDED* to the timeout= value
 // to determine the max time we're willing to wait.
-var BUFFER_PERIOD_MS = 80 * 1000;
+var BUFFER_PERIOD_MS = 5 * 1000;
```

Comment:
initial sync gets the same timeout as incremental sync. So we'd actually be giving 20s to transfer all the data down the pipe (assuming the server responded immediately). Nevertheless this probably does need bumping somewhat. Let's try 30s as a compromise.